lib/utils/kube.go

```diff
@@ -74,7 +85,7 @@ func GetTeamNumber() int {
 }
 
 func GetMongoIP() string {
-	client, err := GetKubeClient()
+	client, err := GetKubeClient("")
```

Comment:
Why are we passing `""` as arguments?

Reply:
It takes the argument for kubectl config file. If you pass an empty string it sets the standard path to the config file in .kube folder.

Reply:
Make it so that if I don't pass anything it uses the standard path, I shouldn't be needing to pass "" as a default argument.